### DISCUSSION TITLE
Issue: Persistent Firebase Auth Credentials After App Deletion (iOS Only)

### DISCUSSION THREAD

**Discussion by maylorsan** [capawesome-team/capacitor-firebase#652] [Q&A]
Hi!

I am using the capacitor-firebase/authentication plugin to handle Firebase Authentication in my Capacitor app. I've encountered an issue where Firebase authentication credentials persist even after the app is deleted and reinstalled on iOS devices. This behavior results in users remaining logged in after they reinstall the app, which is not the desired behavior for our application.

---

### Expected Behavior
When the app is deleted and reinstalled, Firebase authentication credentials should be cleared, ensuring that users are logged out and need to sign in again.

---

### Current Behavior
After deleting and reinstalling the app on iOS, Firebase authentication credentials are still present, and users remain logged in.

---

### Steps to Reproduce
1. Install the app and log in using Firebase Authentication.
2. Delete the app from the iOS device.
3. Reinstall the app from the App Store.
4. Observe that the user is still logged in without having to re-enter credentials.

---

### Possible Solution
A potential solution might involve detecting a fresh installation of the app and clearing Firebase authentication data during the first launch. This could be implemented by storing a unique identifier in UserDefaults. However, this approach would log out all users upon an app update, which is a critical issue for us. Therefore, we haven't found a good solution for this problem.

---

### Additional Context
This issue affects user experience as users expect to be logged out upon app deletion and reinstallation. Ensuring that Firebase auth credentials are cleared in such scenarios would align with user expectations and improve the app's security posture.

We have found this issue to be specific to iOS. Thank you to the author of this plugin for their work on this useful tool.

**Comment by robingenz** [comment]
I close this as duplicate of https://github.com/capawesome-team/capacitor-firebase/discussions/485.
